src/controller/cmcd-controller.ts

```diff
@@ -24,6 +24,7 @@ import type {
   HlsConfig,
   PlaylistLoaderConstructor,
 } from '../config';
+import { CmcdEncodeOptions } from '@svta/common-media-library/cmcd/CmcdEncodeOptions';
```

Comment:
This should be a type import.

```suggestion
import type { CmcdEncodeOptions } from '@svta/common-media-library/cmcd/CmcdEncodeOptions';
```